client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -196,3 +210,38 @@ export class HubConnection {
         };
     }
 }
+
+class Base64EncodedHubProtocol implements IHubProtocol {
+    private wrappedProtocol: IHubProtocol;
+
+    constructor(protocol: IHubProtocol) {
+        this.wrappedProtocol = protocol;
+        this.name = this.wrappedProtocol.name;
+        this.type = ProtocolType.Text;
+    }
+
+    readonly name: string;
+    readonly type: ProtocolType;
+
+    parseMessages(input: any): HubMessage[] {
+        // atob/btoa are browsers APIs but they can be polyfilled. If this becomes problematic we can use
```

Comment:
...we could also not support binary over SSE on node. I think XmlHttpRequest and WebSockets are more important - once we decide about these the atob/btoa replacement will just follow the suit.